code/ui/Hud.cs

```diff
@@ -34,10 +33,14 @@ public static void OnHotReloaded()
             if (Host.IsClient)
             {
                 Local.Hud?.Delete();
+
                 Hud hud = new Hud();
+
                 if (Local.Client.Pawn is TTTPlayer player && player.LifeState == LifeState.Alive)
                 {
                     hud.AliveHudPanel.CreateHud();
+
+                    hud.AliveHudPanel.Effects.OnHotReloaded();
```

Comment:
Can't we create the elements inside of `CreateHud()`?

Reply:
We don't know whether it's hotloaded then :o